packages/dds/tree/src/test/tree/anchorSet.spec.ts

```diff
@@ -190,14 +194,14 @@ describe("AnchorSet", () => {
 
 		checkEquality(anchors.locate(anchor2), path2);
 		applyTestDelta(makeDelta(detachMark, makePath([fieldFoo, 3])), anchors);
```

Comment:
This line (and another one below in the same test) reuses the same `detachId` despite the fact that the node associated with `anchor4` already has this ID. This doesn't make the test fail because `AnchorSet` doesn't currently enforce that different detached nodes have different IDs, but it would be best to avoid this kind of strange behavior in the test since it might make the test fail in the future when it shouldn't.